lighthouse-core/gather/driver.js

```diff
@@ -1081,7 +1088,7 @@ function wrapRuntimeEvalErrorInBrowser(err) {
  * property on window to the end time of the last long task.
  * instanbul ignore next
  */
-function installPerformanceObserver() {
+function registerPerformanceObserver() {
```

Comment:
I think I prefer the driver method and this function to have different names...but I don't feel strongly if you like it better this way.